NGitLab/Impl/RepositoryClient.cs

```diff
@@ -85,6 +87,16 @@ public IEnumerable<Commit> GetCommits(GetCommitsRequest request)
                 lst.Add($"first_parent={Uri.EscapeDataString(request.FirstParent.ToString())}");
             }
 
+            if (request.Since.HasValue)
+            {
+                lst.Add($"since={WebUtility.UrlEncode(request.Since.Value.ToString("s", CultureInfo.InvariantCulture))}");
```

Comment:
NGitLab uses `Uri.EscapeDataString` everywhere to escape query string value. Can you please use it?

Reply:
Sure thing, I will push a change for this shortly.